### PR TITLE
workflow and python script to auto-update subreddit CSS

### PR DESCRIPTION
**Pull request by LinearArray**
Check #4 for more information.

### CODE REVIEW

**Review by jarmos-san — APPROVED** [review]
LGTM! Can you update the Actions like `actions/checkout` and `actions/setup-python` to their latest versions?

**Comment by LinearArray** [comment]
@Jarmos-san done, i have updated the versions :)

**Review by jarmos-san — APPROVED** [review]

**Comment by jarmos-san** [comment]
@Bhupesh-V the PR looks good to me. I think you should take a look once just in case there's something missing in context to the subreddit itself. Other than that it looks good to me for merging.

**Comment by LinearArray** [comment]
I'll recommend to test it once before pushing.

File: .github/workflows/update.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Update Subreddit CSS
+
+on:
+  push:
+    paths:
+      - 'src/main.css'
```
Comment (Bhupesh-V) [inline .github/workflows/update.yml]: Question:

What's the behavior here? Ideally, the behavior we need:

- only trigger when a pull request is opened against main branch AND main.css file has changed

Let me try the script locally once as well

Comment (Bhupesh-V) [reply .github/workflows/update.yml]: Update, only trigger when pull request is successfully merged

**Comment by LinearArray** [comment]
Updated.

**Pull request merged**